packages/core/src/hooks/useSendTransaction.ts

```diff
@@ -34,38 +37,40 @@ export function useSendTransaction(
 ): UseSendTransactionResult {
   const { calls, ...rest } = props;
 
+  let params: AddInvokeTransactionParameters | undefined;
+
+  if (calls) {
+    params = {
+      calls: transformCalls(calls),
+    };
+  }
```

Comment:
```suggestion
  const params = calls ? { calls: transformCalls(calls) } : undefined;
```

That way it should automatically infer the type of params.

Reply:
right! makes sense.